cmd/server/cmd/BUILD

```diff
@@ -31,7 +31,6 @@ go_library(
         "@com_github_grpcecosystem_opentracing//go/otgrpc:go_default_library",
         "@com_github_grpcecosystem_prometheus//:go_default_library",
         "@com_github_istio_api//:mixer/v1",
-        "@com_github_istio_api//:mixer/v1/config",
```

Comment:
Everywhere that we used "@com_github_istio_api//:mixer/v1/config", we also used: "//pkg/config/proto:go_default_library". Since `cfg.proto` is the primary inclusion in both libraries, this should not be necessary.